tests/llm/cluster_extraction_runner.py

```
@@ -0,0 +1,551 @@
+    lines.append("# Cluster Extraction Eval Report")
+    lines.append("")
+    lines.append(f"_Generated: {datetime.now(timezone.utc).isoformat(timespec='seconds')}_")
+    lines.append("")
+    lines.append(
+        f"- **Cases:** {len(cases)}  &nbsp;&nbsp; **Models:** {len(models)}  "
+        f"&nbsp;&nbsp; **Variants:** {len(PROMPT_VARIANTS)}  "
+        f"&nbsp;&nbsp; **Total runs:** {len(rows)}"
+    )
+    lines.append("")
+
+    lines.append("## Summary by model × variant")
```

Ruff (0.15.12)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)